### PR TITLE
[risk=low] Prod cdr version change

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. cdr, public db version update in prod.
2. Data changes: path,is_main columns were added to survey_question_map to display sub question branching logic in surveys.
3. Pr that has the api and data change description: https://github.com/all-of-us/workbench/pull/1784

### CODE REVIEW

**Comment by calbach** [comment]
In the PR description, could you summarize what changes were applied here? Ideally with references to the relevant PRs?

**Comment by SrushtiGangireddy** [comment]
Added. Thanks,

**Comment by SrushtiGangireddy** [comment]
So, would the prod cdr db be named r_2018q4_4 and the public c2018q4_5 ?

**Comment by SrushtiGangireddy** [comment]
Thanks. Merging in.
